ipam/calico-ipam.go

```diff
@@ -97,11 +97,11 @@ func cmdAdd(args *skel.CmdArgs) error {
 		return err
 	}
 
-	workloadID, _, err := utils.GetIdentifiers(args)
+	handleId, err := utils.GetHandleId(args)
 	if err != nil {
 		return err
 	}
-	logger := utils.CreateContextLogger(workloadID)
+	logger := utils.CreateContextLogger(handleId)
```

Comment:
Please can we have a logger that has the workload ID *and* the handle ID in its context?  The container ID is just a random string so the really useful thing is the combination of the two.

Reply:
I'm also not a fan of putting _everything_ in a utils package, what's wrong with `logger := log.WithField("handle", handleID)`  It's just a short and it saves a trip to the other file to see what's going on.